canonical/store-components
# Reduce visible charms on bundle card

## Done
Reduced the number of visible charms on bundle cards

## QA
- Go to https://store-components-77.demos.haus/?path=/story/bundlecard--default-story
- Check that there are only five icons (the images won't show because the paths are relative)
